daphne/src/roles.rs

```diff
@@ -239,148 +238,176 @@ pub trait DapLeader<S>: DapAuthorizedSender<S> + DapAggregator<S> {
         Ok(self.init_collect_job(&collect_req).await?)
     }
 
```

Comment:
Reviewer note: This is copy-pasted from `process()` below. (Similarly for `run_collect_job()` below.)